src/bd2k/util/strings.py

```diff
@@ -2,6 +2,31 @@
 
 import inspect
 
+def to_english(input_list, separator=",", conjuction="and"):
+    """
+    convert list to a more natural and readable string. If given empty list,
+    returns python's default empty string representation: [].
+
+    :param input_list: list of any type, as long as it can be cast to string
+    :param separator: char (or string) to insert between elements in list
+    :param conjuction: string used to connect the final element in list
+    :return: string
+
+    ex:
+    >>> to_english(['a','b','c'])
+    "a, b, and c "
+    """
+    if len(input_list)==0:
+        return str(input_list)
+    if len(input_list)==1:
+        return str(input_list[0])
+    if any(not isinstance(element, str) for element in input_list):
+        input_list = map(str, input_list)
+    last_element=input_list.pop()
+    separator+=" "
```

Comment:
I think as a user of this function I might want the freedom of choice whether I get the space or not. I'd be ok passing separator in as `", "`, i.e. with a trailing space. Same with the conjunction.